packages/flutter_tools/lib/src/build_system/build_system.dart

```diff
@@ -375,7 +375,7 @@ class Environment {
     return Environment._(
       outputDir: outputDir,
       projectDir: projectDir,
-      packageConfigPath: packageConfigPath,
+      packageConfigPath: packageConfigPath ?? '.dart_tool/package_config.json',
```

Comment:
Will this default work in the case that the project is a part of a pub workspace? Should we be using [`findPackageConfigFileOrDefault`](https://github.com/flutter/flutter/blob/2812d4685c3a1c5fb9284d3d9a8e1d13dd0d79d5/packages/flutter_tools/lib/src/dart/package_map.dart#L48) here instead?

Reply:
Or should `packageCofnigPath` stay required and `findPackageConfigFileOrDefault` be used at the call site instead? I don't know yet.

Reply:
Hm, I'm actually not sure why I made this change anymore since we're not creating an `Environment` instance anywhere in this PR that would necessitate this change... I'm going to revert it and update the docs for `packageConfigPath` to not reference a default.